SmartDeviceLink/SDLStreamingMediaLifecycleManager.m

```diff
@@ -518,7 +523,17 @@ - (void)sdl_handleVideoStartServiceAck:(SDLProtocolMessage *)videoStartServiceAc
     // This is the definitive screen size that will be used
     if (videoAckPayload.height != SDLControlFrameInt32NotFound && videoAckPayload.width != SDLControlFrameInt32NotFound) {
         _screenSize = CGSizeMake(videoAckPayload.width, videoAckPayload.height);
-    } // else we are using the screen size we got from the RAIR earlier
+    } else {
+        // If a preferred resolution was set, use the first option to set the screen size
+        if (self.preferredResolutions.count > 0) {
+            SDLImageResolution *preferredResolution = self.preferredResolutions[0];
```

Comment:
`self.preferredResolutions.firstObject` is a tiny change, but I personally like to use it.

Reply:
Fixed